Description: Bump the SONAME due to ABI Breakage in the 1.9.1 release.
Author: Miguel A. Colón Vélez <debian.micove@gmail.com>
Forwarded: yes
Last-Update: 2015-09-15

--- soundtouch-1.9.1.orig/configure.ac
+++ soundtouch-1.9.1/configure.ac
@@ -19,7 +19,11 @@ dnl this program; if not, write to the F
 dnl Place - Suite 330, Boston, MA  02111-1307, USA
 # Process this file with autoconf to produce a configure script.
 
-AC_INIT(SoundTouch, 1.9.0, [http://www.surina.net/soundtouch])
+AC_INIT([SoundTouch], [1.9.1], [http://www.surina.net/soundtouch])
+dnl Default to libSoundTouch.so.$LIB_SONAME.0.0
+LIB_SONAME=1
+AC_SUBST(LIB_SONAME)
+
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR([config/m4])
 AM_CONFIG_HEADER([config.h include/soundtouch_config.h])
--- soundtouch-1.9.1.orig/source/SoundTouch/Makefile.am
+++ soundtouch-1.9.1/source/SoundTouch/Makefile.am
@@ -65,6 +65,8 @@ libSoundTouchMMX_la_CXXFLAGS = $(AM_CXXF
 libSoundTouchSSE_la_CXXFLAGS = $(AM_CXXFLAGS)
 endif
 
+# Modify the default 0.0.0 to LIB_SONAME.0.0
+libSoundTouch_la_LDFLAGS=-version-info @LIB_SONAME@
 
 # other linking flags to add
 # noinst_LTLIBRARIES = libSoundTouchOpt.la
